pydatalab/Pipfile

```diff
@@ -36,6 +35,9 @@ rosettasciio = "*"
 pyjwt = "*"
 invoke = "*"
 paramiko = "*"
+langchain = "*"
+langchain-openai = "*"
+langchain-anthropic = "*"
```

Comment:
This dep isn't necessary for this PR but given the follow-up in #680 there's no harm in adding it now.

Reply:
Its shouldn't be necessary, but removing it caused lots of problems in pipenv for some reason...